src/main/java/frc/robot/subsystems/elevator/ElevatorInputs.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+import org.littletonrobotics.junction.AutoLog;
+
+@AutoLog
+public class ElevatorInputs {
+    public double power;
+    public MutableMeasure<Distance> height;
+    public MutableMeasure<Distance> heightSetpoint;
+    public double acceleration;
```

Comment:
Acceleration should be `MutableMeasure<Velocity<Velocity<Distance>>>`.

Reply:
d55db0dc92ba1f0127d5a86a8b7ba058e9af621c